libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/ThemeSwitcher.java

```diff
@@ -50,6 +51,16 @@ static void setTheme(Context context, AttributeSet attrs) {
     boolean darkThemeEnabled = uiMode == Configuration.UI_MODE_NIGHT_YES;
     updatePreferencesDarkEnabled(context, darkThemeEnabled);
 
+    // Check for custom theme from NavigationLauncher
```

Comment:
What about extracting this block of code into a private method and give it a name based on the comment? This way `setTheme` will be easier to read and understand and comment will become superfluous so it won't be necessary.